scripts-available/CDB_Helper.sql

```diff
@@ -0,0 +1,163 @@
+-- UTF8 safe and lenght aware. Find a unique identifier with a given prefix
+-- and/or suffix and withing a schema.
+CREATE OR REPLACE FUNCTION cartodb._CDB_Unique_Identifier(prefix TEXT, relname TEXT, suffix TEXT, schema TEXT DEFAULT NULL)
+RETURNS TEXT
+AS $$
+DECLARE
+  rec RECORD;
+  usedspace INTEGER;
+  ident TEXT;
+  i INTEGER;
+  origident TEXT;
+  maxlen INTEGER;
+BEGIN
+  maxlen := 63;
+
+  usedspace := 3;
```

Comment:
The suffix `_xx` that can be added at the end if the identifier is already taken.